Source/PagesController.swift

```diff
@@ -172,6 +174,15 @@ extension PagesController : UIPageViewControllerDelegate {
 
 extension PagesController {
 
+  private func setViewController(viewController: UIViewController, atPage page:Int) {
+    pagesDelegate?.pageViewController(self,
+      setViewController: viewController,
+      atPage: page)
+    if let pageChangeHandler = pageChangeHandler {
+      pageChangeHandler(viewController: viewController, page: page)
```

Comment:
Now I use `PresentationController` and need `pagesDelegate` (extended from `PagesController`) to be notified when page changes. But I can't use it, because it's already occupied by `PresentationController` where it's also important to handle page changing. So I'm trying to move out of usage of `pagesDelegate` inside `PresentationController`.